modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -886,9 +886,9 @@ class Editor implements EditorObservable {
   /**
    * Returns true/false if the editor is dirty or not. It will get dirty if the user has made modifications to the contents.
    *
-   * The dirty state is automatically set to true if you do modifications to the content in other
-   * words when new undo levels is created or if you undo/redo to update the contents of the editor. It will also be set
-   * to false if you call editor.save().
+   * The dirty state is automatically set to true if you do modifications to the content in other words if you undo/redo to update
```

Comment:
Maybe?

```
   * The dirty state is automatically set to `true` when the editor content is modified by the user since
   * initialization or the last `editor.save()` call. This includes changes made using undo or redo.
```